governance-exercises/VoteWizard_executive_exercise_02-08-2023/Executive vote - August 2, 2023.md

```diff
@@ -0,0 +1,157 @@
+
+The Governance Facilitators, Dewiz, Sidestream, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+
+- The Enhanced Dai Savings Rate will be activated increasing Dai Savings Rate to 8%, as detailed below.
+- The Spark Protocol Dai Direct Deposit Module (D3M) Maximum Debt Ceiling will be increased, as detailed below.
+- The Harbor Trade Credit (RWA004-A) vault Debt Ceiling will be decreased and will subsequently transition to a soft liquidation state, as detailed below.
+- The New Silver (RWA002-A) RWA Agreement (doc) parameter will be updated, as detailed below.
+- The AVC Member Compensation for the Second Quarter of 2023 will be carried out, as detailed below.
+- A DAO Resolution for Monetalis Clydesdale (RWA007-A) will be approved, as detailed below.
+- **2 million DAI** will be distributed from the Launch Project budget, as detailed below.
+- A Spark Protocol proxy spell will be triggered, as detailed below.
+
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Enhanced Dai Savings Rate (EDSR) Activation
+
+As per this governance  [poll](https://vote.makerdao.com/polling/QmcTRPLx), modifications have been proposed to the Stability Scope to incorporate details of the [Enhanced Dai Savings Rate (EDSR)](https://mips.makerdao.com/mips/details/MIP104#3-2-2-enhanced-dai-savings-rate-edsr-).
+If this executive proposal passes, the DSR will be updated in accordance with these modifications.
+
+- DSR will be increased by 4.81% from 3.19% to **8%**.
+
+Please review the discussion [thread](https://forum.makerdao.com/t/request-for-gov12-1-2-edit-to-the-stability-scope-to-quickly-implement-enhanced-dsr/21405) and the [modifications](https://github.com/makerdao/mips/pull/938/files) to the Stability Scope to help inform your position before voting.
```

Comment:
I probably would have just linked to the MIPs portal again. The changes are nicely self-contained to one additional section, so I think linking the PR is less useful than the portal here. Or given you already linked the portal above, you could just only list the discussion thread here.